src/components/MovieCard/moviecard.css

```diff
@@ -0,0 +1,32 @@
+.MovieCard {
+    display: flex;
+    /*border: solid 1px orange;*/
+    flex-direction: column;
+    font-family: Helvetica, serif;
+    color: #ffffff;
+    text-transform: uppercase;
+    margin-top: 20px;
+    justify-content: flex-start;
+}
+
+.MovieCard > img {
+    width: 330px;
+}
+
+.MovieCard > p {
+    margin-top: 0;
+    margin-bottom: 0;
+}
```

Comment:
Nested selectors are more expensive to calculate. Please introduce appropriate classes for them

Reply:
fixed